stream-compositions/services/payment-order-composition-service/src/main/java/com/backbase/stream/compositions/paymentorders/core/mapper/PaymentOrderMapper.java

```diff
@@ -0,0 +1,49 @@
+package com.backbase.stream.compositions.paymentorders.core.mapper;
+
+import com.backbase.dbs.paymentorder.api.service.v2.model.PaymentOrderPostRequest;
+import com.backbase.dbs.paymentorder.api.service.v2.model.PaymentOrderPostResponse;
+import com.backbase.dbs.paymentorder.api.service.v2.model.UpdateStatusPut;
+import com.backbase.stream.compositions.paymentorder.integration.client.model.PullIngestionRequest;
+import com.backbase.stream.compositions.paymentorders.core.model.PaymentOrderIngestPullRequest;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * This is a mapper for TransactionsPostRequestBody objects used in:
```

Comment:
Comment does not seem to be accurate, nothing about `TransactionsPostRequestBody` here.

Reply:
Copy and paste error. Fixed now